docs/apiSpecifications/Receipt API.yml

```diff
@@ -106,10 +124,28 @@ paths:
                       example: true
   /reference-data/hazardous-property-codes:
     get:
-      description: Endpoint used to get a list of hazardous property code. It returns a code or code list on <b>200</b> success or a <b>401</b> on failure.
+      description: |
+        Endpoint used to get a list of hazardous property codes. It returns a code or code list on <b>200</b> success or a <b>401</b> on failure. <p></p>
+        Example Request
+        ```json
+        curl --request GET \--url https://waste-movement-external-api.api.dev.cdp-int.defra.cloud/reference-data/hazardous-property-codes \
+        --header 'authorization: Bearer eyJraWQiOiJQYnJiZXZvYUF5d1NQcG5KUWlsQXVCT1Q4aVdyNUFcL3RaQkZHaTk5TU5CTT0iLCJhbGciOiJSUzI1NiJ9.eyJzdWIiOiIybHRldXNlaHN
+        ```
       responses:
         "200":
-          description: Success.
+          description: |
+            Success
+            ```json
+            {
+              "code": "HP_1"
+            },
+            {
+              "code": "HP_2"
+            },
+            {
+              "code": "HP_3"
+            }

```

Comment:
This endpoint returns an array of objects, so perhaps this example response should be wrapped in square brackets to indicate that?